docs/components/organization/organization-profile.mdx

```diff
@@ -20,29 +16,55 @@ The `<OrganizationProfile />` component is used to render a beautiful, full-feat
 
 ## Usage
 
-<CodeBlockTabs options={['Next.js', 'React', 'Remix']}>
-```jsx copy filename="/pages/organization-profile/[[...index]].tsx"
-import { OrganizationProfile } from "@clerk/nextjs";
-
-export default function OrganizationProfilePage() {
-  return <OrganizationProfile />;
-}
-```
-```jsx copy filename="organization-profile.tsx"
-import { OrganizationProfile } from "@clerk/clerk-react";
-
-export default function OrganizationProfilePage() {
-  return <OrganizationProfile />;
-}
-```
-```jsx copy filename="/pages/organization-profile/$.tsx"
-import { OrganizationProfile } from "@clerk/remix";
-
-export default function OrganizationProfilePage() {
-  return <OrganizationProfile />;
-}
-```
-</CodeBlockTabs>
+<Tabs items={['Next.js', 'React', 'Remix']}>
+  <Tab>
+    You can embed the `<OrganizationProfile />` component using the [Next.js optional catch all route](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-routes). This allows you to redirect the user inside your application.
+  
+    <CodeBlockTabs options={["Pages router", "App router"]}>
+      ```jsx copy filename="/pages/organization-profile/[[...index]].tsx"
+      import { OrganizationProfile } from "@clerk/nextjs";
+
+      export default function OrganizationProfilePage() {
+        return (
+          <OrganizationProfile routing='path' path="/organization-profile" />
```

Comment:
Based on the snippet here: https://clerk.com/docs/nextjs/organizationprofile

I think these props belong to the App Router snippet.